doc/man/scons.xml

```diff
@@ -997,36 +992,37 @@ options may be specified,
 in which case
 <command>scons</command>
 will read all of the specified files.</para>
-
   </listitem>
   </varlistentry>
+
   <varlistentry>
   <term>-h, --help</term>
   <listitem>
 <para>Print a local help message for this build, if one is defined in
-the SConscript file(s), plus a line that describes the
+the SConscript files, plus a line that describes the
```

Comment:
(s) seems more correct. You could have only one file (SConstruct)

Reply:
I don't have a problem setting this back.  But the manpage does now define the term "SConscript files" as meaning the configuration, whatever number of files it actually is.